### DISCUSSION TITLE
Over $300 for 3D printing the pieces...is that right? Better options?

### DISCUSSION THREAD

**Discussion by ic-racer** [srozum/film_camera_tester#22] [Q&A]
A local 3D printing facility quoted over $300 for the 18 pieces needed. That seems quite a lot of money. Can other's tell me how they are getting the pieces printed and the cost?

Also, the piece called "flexible-hood.stl" is that supposed to be made of flexible plastic or rubber? 

Thank you in advance, 
IC-RACER

**Comment by srozum** [comment]
I cannot answer your question since I printed all parts using my own printer. But take into consideration that some parts are quite big and take 4-8 hours to print, so you probably paying for machine time.

And yes, flexible hood printed with TPU (flexible filament), which requires extra tuning and slow printing.

**Reply by ic-racer** [reply]
Again, thank you for responding. 
I re-contacted them and they initially quoted for commercial fastest delivery. They are going to re-quote me, so hope it is reasonable. 
I'll have to let them know about the TPU for the hood. 

Also, what about this blue piece circled in the picture, I don't think I see it in any of the STL folders:
![enclosure-1](https://user-images.githubusercontent.com/128929440/228041028-f6fd3bab-2864-4a04-9382-05bdcd01530a.jpg)

**Reply by srozum** [reply]
It's not required piece. I'll upload it. And, please, keep separate questions in separate threads.

**Reply by ic-racer** [reply]
Ok I wasn't sure of the QA protocol.

**Reply by polygonfuture** [reply]
@ic-racer You could look into craftcloud 3d printing services:

https://craftcloud3d.com/

You could also try Shapeways if you are U.S. based.  Their versatile nylon is a very cheap print usually, but no idea with the print volumes how much that would come out to for all the parts.

**Reply by fantastecphoto** [reply]
I also printed with my own printer, but you can get the 3d parts printed by JLCPCB when you order the boards. I got the sensor 3d parts printed by them in black resin.

**Reply by ic-racer** [reply]
Thank you for the suggestions!

**Reply by Surreal62** [reply]
Can recommend JLCPCB for quality 3D printing.
I sourced all printed parts needed for the unit and sensors from them for just over £36 delivered to the UK.
